src/Code.gs

```diff
@@ -461,17 +481,29 @@ const ChatGPTApp = (function () {
             .addParameter("prompt", "string", "The question you want to ask the assistant")
             .endWithResult(true);
 
+          if (attachmentIdentificator) {
+            runOpenAIAssistantFunction.addParameter("idAttachment", "string", "the Id of the file attached")
+            runOpenAIAssistantFunction.addParameter("typeOfAttachment", "string", "type of attachment (code_interpreter or file_search)");
+          }
+
           if (numberOfAPICalls == 0) {
 
             tools.push({
               type: "function",
               function: runOpenAIAssistantFunction
             });
 
-            messages.push({
-              role: "system",
-              content: `You can use the assistant ${assistantIdentificator} to retrieve information from : ${vectorStore}`
-            });
+            if (attachmentIdentificator) {
+              messages.push({
+                role: "system",
+                content: `You can use the assistant ${assistantIdentificator} to retrieve information from : ${vectorStore}, with the file "${attachmentIdentificator}" with the type "${typeAttachment}"`
```

Comment:
proposition : 

`You can use the assistant ${assistantIdentificator} to retrieve information from : ${vectorStore}.
Attached, you can use the ${typeAttachment} file: "${attachmentIdentificator}" .`